zachsinterview/src/main/java/com/imprev/zachsinterview/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.imprev.zachsinterview;
+
+public class Main {
+
+    /**
+     * Demonstrates how to connect to thirdPartyAPI and tests how the numerical key store may be used.
+     */
+    public static void main(String[] argv) throws NumericalKeyValueException {
+        
+        String username = "moxiworks";
+        String password = "fjdskal3kd!jdkdInterviewFun!!!";
+        String apiUrl = "https://example.com/thirdPartyApiURL";
```

Comment:
We shouldn't expose credentials directly within the main class/hardcoded. A separate configuration file would be a more secure solution. Then use a credentials provider to retrieve the credentials file and pull the credentials needed.